Translations/pt_BR.po

```diff
@@ -1568,128 +1559,132 @@ msgstr "Pausa"
 
 #: Source/help.cpp:26
 msgid "$Keyboard Shortcuts:"
-msgstr ""
+msgstr "$Atalhos de teclado:"
 
 #: Source/help.cpp:27
 msgid "F1:    Open Help Screen"
-msgstr ""
+msgstr "F1:    Abrir tela de ajuda"
 
 #: Source/help.cpp:28
 msgid "Esc:   Display Main Menu"
-msgstr ""
+msgstr "Esc:   Mostrar menu principal"
 
 #: Source/help.cpp:29
 msgid "Tab:   Display Auto-map"
-msgstr ""
+msgstr "Tab:   Mostrar automapa"
 
 #: Source/help.cpp:30
 msgid "Space: Hide all info screens"
-msgstr ""
+msgstr "Espaço: Ocultar todas as janelas de informação"
 
 #: Source/help.cpp:31
 msgid "S: Open Speedbook"
-msgstr ""
+msgstr "S: Abrir livro da velocidade"
```

Comment:
This is really bad. Suggestion:
```suggestion
msgstr "S: Abrir livro-rápido"
```
(still a bit bad but less so IMHO)

Reply:
@julealgon this one is hard, maybe:
"Abrir livro de atalhos"?

Reply:
Yeah... its tough because "speedbook" is a nonexistent compound word and this concept (of combining arbitrary words and forming new words) doesn't really exist in portuguese.

I think your suggestions are more intuitive indeed.

Reply:
Could you update the suggestion so that I can add it?

Reply:
Done

Reply:
Ubuntudanmark.dk you know you could have put it here so that it would be clearly connected to the conversation 😅